exporter/stats/stackdriver/stackdriver.go

```diff
@@ -87,6 +87,13 @@ type Options struct {
 	// can be buffered before batch uploading them to the backend.
 	// Optional.
 	BundleCountThreshold int
+
+	// Resource is the Stackdriver MonitoredResource representing a
```

Comment:
Just a doc suggestion, could we spell out upfront that Resource is optionally, say:
```go
// Resource is an optional field that represents the Stackdriver MonitoredResource
```

Reply:
Sure, done.